mantle/cmd/kolet/kolet.go

```diff
@@ -43,6 +43,23 @@ const (
 // kolaRebootStamp should be created by tests that want to reboot
 const kolaRebootStamp = "/run/kola-reboot"
 
+// These are defined by https://salsa.debian.org/ci-team/autopkgtest/raw/master/doc/README.package-tests.rst
+const (
+	autopkgTestRebootPath   = "/tmp/autopkgtest-reboot"
```

Comment:
Hmm, why not e.g. `/usr/local/bin`? And that way we don't have to re-write it on each boot either.

Reply:
That path is part of the autopkgtest specification.